shiny/input_handler.py

```diff
@@ -48,7 +48,7 @@ def _process_value(self, type: str, value: Any, name: str, session: Session) ->
 to the ``input`` argument of an :func:`~shiny.App`'s ``server`` function.
 
 The ``type`` is based on the ``getType()`` JavaScript method on the relevant Shiny
-input binding. See `this article <https://shiny.rstudio.com/articles/js-custom-input.html>`_
+input binding. See `this article <https://shiny.posit.co/articles/js-custom-input.html>`_
```

Comment:
This is a post from 2017 about doing this in the R package. IIUC the JavaScript parts would still be the same between Python and R, but the Python part would be different. 

1. Is this article still accurate even for R?
2. Assuming it should be linked here, we should note that the article is useful for the JS part, and the Python binding is analogous to the R code, right?

Reply:
1. The article is still accurate for R.
2. That sounds good to me.

As an aside, the formatting on that particular page got messed up from the Jekyll->Quarto transition.
https://github.com/rstudio/shiny-dev-center/issues/240